app/views/posts/show.html.slim

```diff
@@ -12,6 +12,6 @@
       = @post.content
     hr
     span
-      - if @owner
+      - if user_signed_in? && current_user.id == @post.user_id
```

Comment:
Very good!

Bet posts_helper tagad tu nekur neizmanto un pagaidam vari to dzēst